crates/autopilot/src/run_loop.rs

```diff
@@ -230,8 +242,8 @@ impl RunLoop {
                         owner: order.metadata.owner,
                         partially_fillable: order.data.partially_fillable,
                         executed: Default::default(),
-                        pre_interactions: Default::default(),
-                        post_interactions: Default::default(),
+                        pre_interactions: map_interactions(&order.interactions.pre),
+                        post_interactions: map_interactions(&order.interactions.post),
```

Comment:
This file has the only logic changes, just trivially marshaling auction data to the co-located solvers.